Sources/SwiftDocC/Model/Rendering/RenderNode/CodableRenderReference.swift

```diff
@@ -37,14 +37,12 @@ struct CodableRenderReference: Codable {
             reference = try TopicRenderReference(from: decoder)
         case .section:
             reference = try TopicRenderReference(from: decoder)
-        case .download:
+        case .download, .externalLocation:
```

Comment:
Not sure if this is important, but IIUC the line below won't work because `DownloadReference(from: Decoder)` re-decodes the `type` key?

Reply:
Right, i edited this method because i didn't want to introduce a deprecation warning to our builds. I can put it back if that's what y'all would prefer.